packages/regl-worldview/src/commands/GLText.js

```diff
@@ -297,7 +297,11 @@ function makeTextCommand(alphabet?: string[]) {
     const atlasTexture = regl.texture();
     const makeDrawText = (isHitmap: boolean) => {
       return regl({
-        depth: defaultDepth,
+        // When using scale invariance, we want the text to be drawn on top
+        // of other elements. This is achieved by disabling depth testing
+        // In addition, make sure the <GLText /> command is the last one
+        // being rendered.
+        depth: command.scaleInvariant ? disabledDepth : defaultDepth,
```

Comment:
Curious if you have considered using `layerIndex`?

Reply:
Disabling depth testing is required regardless of the rendering order. Then, `layerIndex` should help for the second part of my comment, where we need to make sure `<GLText />` is the last command to be rendered, but I preferred not to enforce it on the command itself. What do you think?

Reply:
👍 Makes sense